### PR TITLE
Updating cloudant trigger package doc.

### PR DESCRIPTION
**Pull request by fxulusoy**

### CODE REVIEW

File: docs/catalog.md [hunk 1]
```diff
@@ -103,7 +103,11 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
 
 ### Listening for changes to a Cloudant database
 
-You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database.
+You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database. The parameters are as follows:
+
+- `dbname`: Name of cloudant database.
+- `includeDoc`: If set to true, each trigger event that is fired includes the modified Cloudant document. 
+- `maxTriggers`: Stop firing triggers when this limit is reached. Defaults to 1000.
```
Comment (jeremiaswerner) [inline docs/catalog.md]: pls also document the maximum of 10.000. Its missed in the alarms package as well.

File: docs/catalog.md [hunk 2]
```diff
@@ -103,7 +103,11 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
 
 ### Listening for changes to a Cloudant database
 
-You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database.
+You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database. The parameters are as follows:
+
+- `dbname`: Name of cloudant database.
```
Comment (rabbah) [inline docs/catalog.md]: Cloudant (capital C).

File: docs/catalog.md [hunk 3]
```diff
@@ -103,7 +103,11 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
...
+- `includeDoc`: If set to true, each trigger event that is fired includes the modified Cloudant document. 
+- `maxTriggers`: Stop firing triggers when this limit is reached. Defaults to 1000. You can set it to maximum 10.000. If you try to set more than 10.000, the request is rejected.
```
Comment (rabbah) [inline docs/catalog.md]: `.` -> `,` in 10.000

**Comment by markusthoemmes** [comment]
@csantanapr thoughts?

**Comment by csantanapr** [comment]
@markusthoemmes LGTM 👍

**Comment by fxulusoy** [comment]
I have rebased the branch. It is ready to merge.

**Comment by markusthoemmes** [comment]
merged as 0ce8e8fe2eb4a30c4b7b13637f122126ce6db268

**Pull request closed without merging**